machine-learning-scikit-learn.ipynb

```diff
@@ -0,0 +1,744 @@
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's wrap it with `dask_ml.wrappers.Incremental`. `Incremental` is a meta-estimator, or an estimator that takes in another estimator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_ml.wrappers import Incremental\n",
+    "\n",
+    "inc = Incremental(est, scoring='accuracy')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`Incremental` does data management: it calls `est.partial_fit` on each chunk of the passed data. Dask moves the model to different chunks of the data (or vice versa) to let this happen. Note: Dask-ML `Incremental` gives data to Scikit-learn and does not change the underlying optimization algorithm that Scikit-learn uses. There are different optimization algorithms in Dask-ML that do this, especially for clustering and matrix decomposition.\n",
+    "\n",
+    "Our model (SGDClassifier here) must implement `partial_fit` to be used with `Incremental`. A list of models that implement this API is available on a scikit-learn documentation page under \"Incremental learning\": http://scikit-learn.org/stable/modules/scaling_strategies.html#incremental-learning\n",
+    "\n",
+    "It is important to specify the scoring parameter in `Incremental`; otherwise, scikit-learn scorers are fed Dask arrays (which they're not optimized for).\n",
+    "\n",
+    "\n",
+    "[1]:http://scikit-learn.org/stable/modules/scaling_strategies.html\n",
+    "[inc]:http://scikit-learn.org/stable/modules/scaling_strategies.html#incremental-learning"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Of course, `Incremental` implements a `fit` method, which will perform one loop over the dataset, calling `partial_fit` over each chunk in the Dask array."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Incremental(estimator=SGDClassifier(alpha=0.0001, average=True, class_weight=None, epsilon=0.1,\n",
+       "       eta0=0.0, fit_intercept=True, l1_ratio=0.15,\n",
+       "       learning_rate='optimal', loss='log', max_iter=None, n_iter=None,\n",
+       "       n_jobs=1, penalty='l1', power_t=0.5, random_state=None,\n",
+       "       shuffle=True, tol=0.001, verbose=0, warm_start=False),\n",
+       "      scoring='accuracy')"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "classes = da.unique(y).compute()\n",
+    "inc.fit(X_train, y_train, classes=classes)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's call `partial_fit` multiple times.\n",
```

Comment:
I would remove this block, since implementing `max_iter` for Incremental will I think make it obsolete?

Reply:
I want to leave this block we're show casing `Incremental`, which will very likely be used for other tools to get some idea or progress/etc. I would imagine that the `data` dictionary I use will be used, and probably with more data.

I have restructered a bit, to move the persisting data section below the "calling `partial_fit`" header.